livekit-agents/livekit/agents/llm/llm.py

```diff
@@ -52,12 +52,16 @@ class FunctionToolCall(BaseModel):
     name: str
     arguments: str
     call_id: str
+    extra_content: dict[str, Any] | None = None
```

Comment:
I would still name this `extra` IMO
```suggestion
    extra: dict[str, Any] | None = None
```